core/src/main/kotlin/com/imashnake/animite/core/extensions/MarkdownStyleExt.kt

```diff
@@ -0,0 +1,42 @@
+package com.imashnake.animite.core.extensions
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.boswelja.markdown.style.BlockQuoteStyle
+import com.boswelja.markdown.style.CodeBlockStyle
+import com.imashnake.animite.core.R
+
+@Composable
+fun animiteBlockQuoteStyle(
+    background: Color = MaterialTheme.colorScheme.surfaceVariant,
+    barWidth: Dp = dimensionResource(R.dimen.markdown_block_corner_radius),
+    barColor: Color = MaterialTheme.colorScheme.onSurfaceVariant,
+    barShape: Shape = RectangleShape,
+    innerPadding: PaddingValues = PaddingValues(8.dp)
+) = BlockQuoteStyle(
+    background = background,
+    shape = RoundedCornerShape(barWidth),
+    barShape = barShape,
+    barWidth = barWidth,
+    barColor = barColor,
+    innerPadding = innerPadding
+)
+
+@Composable
+fun animiteCodeBlockStyle(
+    background: Color = MaterialTheme.colorScheme.surfaceVariant,
+    shape: Shape = RoundedCornerShape(dimensionResource(R.dimen.markdown_block_corner_radius)),
```

Comment:
Why are we passing a Dp in one and a Shape in the other

Reply:
Markdown code blocks in Animite will have a shape that is determined by the width of the bar: 

<img width="359" alt="Screen Shot 2024-02-27 at 4 28 47 AM" src="https://github.com/imashnake0/Animite/assets/56754616/8911a422-861c-44c8-97bb-5a0be4db44df">

(Yes, I take you guys very seriously 👽)